templates/_macros/quote-wrapper.jinja

```diff
@@ -0,0 +1,172 @@
+    <p class="p-heading--{{ quote_heading_level }}">
+      <i>
+        &#8220;{{ quote_text }}&#8221;
+      </i>
+    </p>
+  {%- endmacro %}
+
+  {%- macro _citation_block() -%}
+    {%- if has_citation -%}
+      <!--Optional citation block-->
+      <p>
+        {% if has_citation_source_name -%}
+          <!--Optional citation source name-->
+          <strong>{{ citation_source_name_text }}</strong>
+          {% if has_citation_source_title or has_citation_source_organisation -%}
+            <!--If the citation name is followed by title and/or organisation, add a line break-->
+            <br>
+          {% endif -%}
+        {% endif -%}
+        {% if has_citation_source_title or has_citation_source_organisation -%}
+          <!--Optional citation source title and/or organisation-->
+          <span class="u-text--muted">
+            {% if has_citation_source_title -%}
+              <!--Optional citation source title-->
+              {{ citation_source_title_text }}
+              {%- if has_citation_source_organisation %}
+                <!--Add a line break between the title and org if both are present-->
+                <br>
+              {%- endif %}
+            {% endif %}
+            {%- if has_citation_source_organisation -%}
+              <!--Optional citation source organisation-->
+              {{ citation_source_organisation_text }}
+            {%- endif %}
+          </span>
+        {%- endif %}
+      </p>
+    {% endif %}
+  {%- endmacro -%}
+
+  {%- macro _heading_block() %}
+    {% if has_heading_row -%}
+      <!--Optional heading -->
+      <div class="p-section--shallow">
+        <hr class="is-highlighted is-fixed-width">
+        <div class="row">
+          {%- if has_title %}
+            <!--Optional heading text-->
+            <div class="col-3 col-medium-3">
+              <div class="p-section--shallow">
+                <p class="p-muted-heading">{{ title_text }}</p>
+              </div>
+            </div>
+          {%- endif -%}
+
+          {%- if has_heading_link %}
+            <!--Optional heading link -->
+            <div class="col-3 col-medium-3 col-start-large-10 col-start-medium-5">
+              <div class="p-section--shallow">
+                <p class="p-text--default">
+                  {{ heading_link_content }}
+                </p>
+              </div>
+            </div>
+          {% endif -%}
+        </div>
+      </div>
+    {% endif -%}
+  {%- endmacro %}
+
+  <div class="p-section">
+    {{- _heading_block() -}}
+    <div class="row--25-75-on-large">
+      {% if has_signpost_image -%}
+        <!--Optional signpost image-->
+        <div class="col">
+          <div class="row">
+            <div class="col-2 col-medium-2 col-small-2">
+              {{ signpost_image_content }}
+            </div>
+          </div>
+        </div>
+      {% endif -%}
+      {#- Skip to fourth column if the signpost_image (which takes up first three columns) is missing -#}
```

Comment:
Note to self tomorrow: this comment can be removed, as we're now using the 25/75 row, making this obsolete

Reply:
Updated